modules/access-solution/pages/rest-requests.adoc

```diff
@@ -10,15 +10,16 @@ Here is a step-by-step guide to generating and using an authorization token for
 
 === 1. Navigate to User Management
 
-From GraphStudio, go to *Admin Portal*, and click *User Management*.
+From GraphStudio, go to *Admin Portal* and click *User Management*.
```

Comment:
Please use the menu:[] macro for navigation.

From GraphStudio, go to menu:Admin Portal[User Management]

https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/